File: code1/code1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for x in range(len(nums)):
            for y in range(x+1, len(nums)):
                #print("%d %d" %(nums[x],nums[y]))
                if (nums[x]+nums[y]) == target:
                    return [x,y]
        pass

File: code1/code1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> re_arr;
        for (int i = 0; i < nums.size(); i++)
        {
            for (int j = i+1; j < nums.size(); j++)
            {
                //cout << nums[i] << " " << nums[j] << endl;
                if (nums[i]+nums[j] == target)
                {
                    re_arr.push_back(i);
                    re_arr.push_back(j);
                    return re_arr;
                }
            }
        }
        return re_arr;
    }
};